pyproject.toml

```diff
@@ -136,7 +136,11 @@ test-command = [
 profile = "black"
 
 [tool.mypy]
+disallow_untyped_defs = true
+exclude = "^((tests|docs|examples|bin)/|conftest.py|src/clib/|src/xtgeo/(common|cube|xtgeo|grid3d|metadata|plot|roxutils|surface|well|xyz)/?)"
```

Comment:
The idea is as we go and get more and more converge with mypy we remove modules from the exclude "list".